modules/nw-sriov-installing-operator.adoc

```diff
@@ -158,5 +158,5 @@ $ oc annotate ns/openshift-sriov-network-operator workload.openshift.io/allowed=
 +
 [NOTE]
 ====
-For the Single Node OpenShift (SNO) cluster, the annotation `workload.openshift.io/allowed=management` is required for the namespace.
+For the {sno} cluster, the annotation `workload.openshift.io/allowed=management` is required for the namespace.
```

Comment:
You could change "For the `{sno}` cluster" to "For a `{sno}` cluster". Alternatively, you could replace it with "For `{sno}` clusters "